src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -12789,6 +12790,64 @@ await VerifyItemExistsAsync(
             matchingFilters: [FilterSet.MethodFilter, FilterSet.TargetTypedFilter]);
     }
 
+    [InlineData("IFoo", new string[] { "Foo", "FooDerived", "FooGeneric" })]
+    [InlineData("IFoo[]", new string[] { "IFoo", "IFooGeneric", "Foo", "FooAbstract", "FooDerived", "FooGeneric" })]
+    [InlineData("IFooGeneric<int>", new string[] { "FooGeneric" })]
+    [InlineData("IFooGeneric<int>[]", new string[] { "IFooGeneric", "FooGeneric" })]
+    [InlineData("IOther", new string[] { })]
+    [InlineData("Foo", new string[] { "Foo" })]
+    [InlineData("FooAbstract", new string[] { "FooDerived" })]
+    [InlineData("FooDerived", new string[] { "FooDerived" })]
+    [InlineData("FooGeneric<int>", new string[] { "FooGeneric" })]
+    [InlineData("object", new string[] { "C", "Foo", "FooDerived", "FooGeneric" })]
+    [Theory, Trait(Traits.Feature, Traits.Features.TargetTypedCompletion)]
+    public async Task TestTargetTypeCompletionInCreationContext(string targetType, string[] expectedItems)
+    {
+        ShowTargetTypedCompletionFilter = true;
+
+        var markup =
+            $$"""
```

Comment:
Can't use foo because of policheck.  Use Goo

Reply:
Changed, weird though as other tests in this file use that name.